Source/WebCore/css/calc/CSSCalcTree+Evaluation.h

```diff
@@ -31,6 +31,10 @@ namespace WebCore {
 
 class CSSCalcSymbolTable;
 
+namespace Style {
+class BuilderState;
```

Comment:
This was probably from an earlier iteration where you passing this in explicitly instead of in the conversion data? Probably not needed now?

Reply:
Yeah, tried a different approach first.